deploy/kiali/kiali_cr.yaml

```diff
@@ -353,7 +360,7 @@ spec:
 #    ---
 #    tolerations: []
 #
-# Determines which priority levels of log messages Kiali will output.
+# DEPRECATED - Determines which priority levels of log messages Kiali will output.
```

Comment:
As part of the deprecated message, I would point the reader to the new setting. Something like this:
```suggestion
# DEPRECATED - use the logger.log_level setting.
# Determines which priority levels of log messages Kiali will output.
```